pkgs/native_assets_builder/test_data/native_add_v1_0_0/build.dart

```diff
@@ -1,32 +0,0 @@
-// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Maybe we should have a v1_3_0 test, but we should have already had that instead of forcing you to write it in this PR. 🤓 

So feel free to ignore this comment.

Reply:
There's already plenty of tests in `pkgs/native_assets_builder/test_data` that should give test coverage (they just don't have any version like `v1_x_y` in their name). I wouldn't know what test to add that is *different* from the ones that are already there.